src/modules/form/components/client/phones/MultiPhoneInput.tsx

```diff
@@ -7,7 +7,7 @@ import { PhoneInputType } from '../types';
 import { useRenderLastUpdated } from '../useRenderLastUpdated';
 import { createInitialValue } from '../util';
 
-import PhoneInput from './PhoneInputGroup';
+import PhoneInputGroup from './PhoneInputGroup';
```

Comment:
I'm not sure if this is what had happened here, but I noticed that when I asked my IDE to auto-import `PhoneInput`, it imported it from `PhoneInputGroup` but renamed as `PhoneInput` -- which led to my confusion of thinking that the phone type dropdown and notes text field are built-in to the `PhoneInput` component, when they are actually not, they are only built-in to the `PhoneInputGroup` component.

Anyway, when I searched the codebase for usages of `PhoneInput` I noticed the same import-rename shenanigans going on here, so I replaced it for more clarity -- let me know if you think this is an ok change to make.

Reply:
thanks! yeah thats great